.github/workflows/firebase-qa-pull-request.yml

```diff
@@ -30,6 +30,7 @@ jobs:
           ls -la
       - name: Build
         run: |
+          export NODE_OPTIONS="--max_old_space_size=4096"
```

Comment:
Huh. I didn't know about that option. Looks like "old space" is for garbage collected heap storage.

https://stackoverflow.com/questions/48387040/how-do-i-determine-the-correct-max-old-space-size-for-node-js/48392705#48392705

4GiB seems like a lot... Is this for the preview links?